Library/Services/INfieldSurveyLanguageTranslationsService.cs

```diff
@@ -0,0 +1,77 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Nfield.Models;
+using Nfield.SDK.Models;
+
+namespace Nfield.Services
+{
+    /// <summary>
+    /// Represents a set of methods to read and update survey languages.
+    /// </summary>
+    public interface INfieldSurveyLanguageTranslationsService
+    {
+        #region CRUD on Language Translations
+
+        /// <summary>
+        /// Gets all languages and their translations for a survey, as a queryable object.
+        /// </summary>
+        /// <param name="surveyId">The survey for which language names and ids are returned</param>
+        /// <exception cref="T:System.AggregateException"></exception>
```

Comment:
We should probably remove all the empty comments in the file about the exceptions (or fill them)

Reply:
Most of the interfaces have these listed, I stuck with it.